python/aitemplate/compiler/ops/conv/conv2d.py

```diff
@@ -505,22 +504,38 @@ def _profile_single_workload(self, profiler_prefix, exec_key, devices, force_cac
                 f"{op_type} {exec_entry_sha1}.\n",
                 "Please adjust target.select_minimal_algo function.",
             )
-
-        profiler_filename = get_profiler_filename(self._attrs, "conv")
-        runner = backend.profiler_runner.Runner(
-            devices, self._attrs["name"], timeout=180
-        )
-        x_shape = self._invert_exec_key(exec_key)
-        command = self._gen_profile_cmd(profiler_prefix, profiler_filename, x_shape)
-        runner.push(profiler_filename, command)
+        if target.name() == "rocm":
```

Comment:
This makes codebase a bit hard to maintain. Do you want to also refactor rocm profiler codegen part so that rocm / cuda are consistent?

Reply:
The feature will be done in the next PR.